src/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs

```diff
@@ -71,6 +63,79 @@ public void SslStream_StreamToStream_Authentication_IncorrectServerName_Fail()
             }
         }
 
+        [Fact]
+        public void SslStream_StreamToStream_Back2Back_ClientWrite_Success()
+        {
+            SslStream clientSSLStream = null;
+            SslStream ServerSSLStream = null;
+
+            try
+            {
+                GetServerClientSSLStream(out clientSSLStream, out ServerSSLStream);
+                clientSSLStream.Write(sampleMsg);
+                clientSSLStream.Write(sampleMsg);
+            }
```

Comment:
all we needed to verify that calling write >=twice does not throw net_io_invalidnestedcall exception. so there is no assert/check required here.